docs/tutorials/_index.md

```diff
@@ -3,4 +3,62 @@ title: "Tutorials"
 linkTitle: "Tutorials"
 weight: 65
 type: docs
+no_list: true
 ---
+
+{{< alert title="Tip" color="tip" >}}
+While tutorials may list specific hardware, the hardware only affects the configuration of a project.
```

Comment:
```suggestion
While following a tutorial may require specific hardware, the hardware requirements only affect the part of a project where you configure your components.
```
(opt)  Wording is a little big confusing, thought this could improve